src/main/java/seedu/billboard/model/tag/Tag.java

```diff
@@ -32,6 +32,10 @@ public static boolean isValidTagName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public String getTagName() {
+        return tagName;
+    }
+
```

Comment:
isnt tagName public?

Reply:
I removed the method and changed my code access the variable directly.